apps/judicial-system/backend/src/app/modules/notification/services/caseNotification/caseNotification.service.ts

```diff
@@ -579,6 +528,24 @@ export class CaseNotificationService extends BaseNotificationService {
     }
   }
 
+  private getCourtDateCalendarInvite = (
```

Comment:
This is for example a function which is now duplicated in three notification service: case, indictmentCase, defendant. 

As mentioned in the description, this would be an example of a method we could include in a common notification service that case, indictment, defendant notification services could inherit from.

Reply:
The duplicated methods are: 
- getCourtDateCalendarInvite
- uploadEmailToCourt
-